README.md

```diff
@@ -1,4 +1,18 @@
-# AsyncAPI React Component
+<h5 align="center">
+  <br>
+  <a href="https://www.asyncapi.org"><img src="./.github/assets/logo.png" alt="AsyncAPI logo" width="200"></a>
+  <br>
+  React Component
+</h5>
+<h4 align="center">A React component for AsyncAPI specification</h4>
+
+---
+
+> :warning: This package doesn't support AsyncAPI 1.x anymore. We recommend to upgrade to the latest AsyncAPI version using the [AsyncAPI converter](https://github.com/asyncapi/converter). If you need to convert documents on the fly, you may use the [Node.js](https://github.com/asyncapi/converter) or [Go](https://github.com/asyncapi/converter-go) converters.
```

Comment:
imho this is not a big deal therefore separate warning exposed like that doesn't make much sense to me.
after `overview` we should just have a section called like `supported versions` and explain that this component supports 2.0, that people should convert and that is it,
mentioning old version of the component doesn't make much sense as we will not maintain it anyway